book/getting-started/vhost.rst

```diff
@@ -109,9 +109,19 @@ The Nginx configuration could look something like.
       }
   }
 
-
 .. warning::
-  Be sure to also configure your local host-file, if running Sulu locally.
+    Be sure to also configure your local host-file, if running Sulu locally.
 
-Now, your Sulu instance is ready.
+MAMP Pro
+--------
+
+In general you should configure your vHost like the `Apache 2.2`_ paragraph above describes it.
 
+If you want to enable the dev-environment (*including the debug toolbar*) you have to be sure that the
+vHost development parameter (*SetEnv SYMFONY_ENV dev*) is set properly
```

Comment:
Well I was talking about the functionality of this variable but u r right in Apache/Sulu scope it's of couz an environment variable.